### ISSUE TITLE
Tabbing (Accessibility)

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#14]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/14) by Katelyn Adams (Bitbucket: [paesh1997](https://bitbucket.org/paesh1997), GitHub: [paesh1997](https://github.com/paesh1997)).**

----------------------------------------

When filling out labor status forms, submitters cannot press 'tab' to move from one field to the next, which is an important accessibility issue.

**Comment by bryantal** [comment]
**Original comment by Bria Williams (Bitbucket: [BriaWilliams1202](https://bitbucket.org/BriaWilliams1202), ).**

----------------------------------------

The new  Labor Status Forms interface has tabbing that works.
